src/transaction/client.rs

```diff
@@ -165,16 +174,18 @@ impl Client {
         self.pd.clone().get_timestamp().await
     }
 
-    /// Cleans MVCC records whose timestamp is lower than the given `timestamp` in TiKV.
+    /// Trigger garbage collection (GC) of the TiKV cluster.
```

Comment:
I think "trigger" might somewhat mislead users. It's asynchronous and TiKV decides when to do actual GC.